firstvoices/backend/search/utils/search_term_query.py

```diff
@@ -0,0 +1,204 @@
+from elasticsearch_dsl import Q
+
+BASE_BOOST = 1.0  # default value of boost
+EXACT_MATCH_PRIMARY_BOOST = 5
+EXACT_MATCH_SECONDARY_BOOST = 4
+FUZZY_MATCH_PRIMARY_BOOST = 3
+FUZZY_MATCH_SECONDARY_BOOST = 2
+EXACT_MATCH_OTHER_BOOST = 1.5
+FUZZY_MATCH_OTHER_BOOST = BASE_BOOST
+
+
+def get_search_term_query(search_term, domain):
+    """
+    All model fields are mapped to 6 full text searchable fields, named as:
+
+    primary_language_search_fields
+    primary_translation_search_fields
+    secondary_language_search_fields
+    secondary_translation_search_fields
+    other_language_search_fields
+    other_translation_search_fields
+
+    Fields are divided based on their importance, and domain (i.e. "language" or "translation").  The "both" domain
+    contains a sum of individual fields present in the other 2 domains.
+
+    Each field has a different boost value, such as exact match of primary fields have higher boost than
+    exact match of secondary fields or fuzzy match of primary fields.
+    Refer to,
+    https://firstvoices.atlassian.net/wiki/spaces/FIR/pages/275251209/ElasticSearch#Consistent-search-across-models
```

Comment:
Can you please add a ticket for 1.0 to put this info in a readme in the repo instead of on a private wiki page?